src/main/modules/expense/components/AddExpense.jsx

```diff
@@ -0,0 +1,35 @@
+import { IoMdClose } from "react-icons/io";
+import { SubTitleWhite } from "../../../../styledComponents";
+import AddExpenseFormm from "./AddExpenseForm";
+
+const AddExpense = ({ handleClick, isExpensePopUpActive }) => {
+  return (
+    <>
+      <div
+        className={`w-screen h-screen bg-tertiary bg-opacity-70 fixed top-0 left-0 z-[999] flex items-center justify-center ${
+          isExpensePopUpActive ? "block" : "hidden"
+        }`}
+        onClick={(e) =>
+          e.target.classList.contains("w-screen") ? handleClick() : ""
```

Comment:
use null instead of empty string here
e.target.classList.contains("w-screen") ? handleClick() : null

Reply:
done